tracking.go

```diff
@@ -67,9 +74,13 @@ func (p *recvdPackets) packetSetReceived(pn uint64, protected bool, nonAcks bool
 
 func (p *recvdPackets) packetSetAcked2(pn uint64) {
 	p.log(logTypeAck, "Setting packet acked2=%v", pn)
-	pk, ok := p.packets[pn]
-	assert(ok)
-	pk.acked2 = true
+	if pn >= p.minNotAcked2 {
+		pk, ok := p.packets[pn]
+		//assert(ok)
```

Comment:
no, it doesn't have to. If a packet has been acked2ed before it might have been removed from the map before. The assert was probably still there because removing the packets is something I added. I removed the assert.